src/js/components/date-picker.js

```diff
@@ -873,11 +872,10 @@ const enhanceDatePicker = (el) => {
 
   internalInputEl.setAttribute("aria-hidden", "true");
   internalInputEl.setAttribute("tabindex", "-1");
-  internalInputEl.classList.add(
-    "usa-sr-only",
-    DATE_PICKER_INTERNAL_INPUT_CLASS
-  );
-  internalInputEl.id = "";
+  internalInputEl.style.display = "none";
```

Comment:
Removed the `sr-only` class and hiding it via `display: none` instead. This resolves an ANDI issue that was flagged about hidden focusable elements.

**Error USWDS 2.11.2**
![image](https://user-images.githubusercontent.com/3385219/121054246-c8e39a00-c781-11eb-9edc-c01aa73580b4.png)